src/node_crypto.cc

```diff
@@ -2789,24 +2789,20 @@ void SSLWrap<Base>::ExportKeyingMaterial(
 
   AllocatedBuffer out = env->AllocateManaged(olen);
 
-  ByteSource key;
-
-  int useContext = 0;
-  if (!args[2]->IsNull() && Buffer::HasInstance(args[2])) {
-    key = ByteSource::FromBuffer(args[2]);
-
-    useContext = 1;
-  }
+  ByteSource context;
+  bool use_context = !args[2]->IsUndefined();
+  if (use_context)
+    context = ByteSource::FromBuffer(args[2]);
 
   if (SSL_export_keying_material(w->ssl_.get(),
                                  reinterpret_cast<unsigned char*>(out.data()),
                                  olen,
                                  *label,
                                  label.length(),
                                  reinterpret_cast<const unsigned char*>(
-                                   key.get()),
-                                 key.size(),
-                                 useContext) != 1) {
+                                   context.get()),
```

Comment:
```suggestion
                                     context.get()),
```

(I usually wouldn’t comment for something as trivial as this, but it seems to fit in with the general idea behind the PR :))

Reply:
Thank you! :)

Reply:
@tniessen in case you don't know: `make format-cpp` before `git commit` should take care of style issues